content/articles/web3js-vs-ethersjs/index.md

```diff
@@ -0,0 +1,206 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /ethers-vs-web3/
+title: Ethers vs web3
+description: This article will help the reader learn the difference between the ethers library and the web3js library.
+author: Chigozie Oduah
+date: 2022-04-5T00:00:00-02:30
+topics: [Languages]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /ethers-vs-web3/hero.jpg
+    alt: Ethers vs web3 Hero Image
+---
+The web3js library and the ethers library are the major libraries for utilizing the Ethereum network in JavaScript. The two have different ways they interact with smart contracts and make transactions on the network. This difference leads to one handling some type of tasks better than the other. In this article, you will learn their differences, their best use cases, and how they both interact with the network.
+### Prerequisites
+To get the full benefits of this article, you need to have the following.
+- You need to have a working knowledge of JavaScript.

```

Comment:
No need to include the highlighted parts below:
- `You need` to have a working knowledge of JavaScript.
- `You should` have a basic understanding of Web3, Smart Contracts, and Blockchain.